Guide_1_Send_Payment.md

```diff
@@ -50,23 +68,67 @@ A detailed description of each field of the payment resource is available in the
 
 ### Create the Submission Resource
 
-The next step is to send the payment by creating a payment submission resource. Note that you have to provide the payment ID in the call.
-
 ```json
 Code for submission resource here!
 ```
 
+The next step is to send the payment by creating a payment submission resource. Note that you have to provide the payment ID in the call.
 
+And just like that, your payment is on its way!
 
 ## Track the Payment Manually
 
-...
+_**(Question: Should Audits be mentioned here? Since it said in the email that audits are not recommended and this is a getting started guide, maybe it's more straightforward and less confusing not to mention it?)**_
```

Comment:
I think it would be useful to use the audit service to view the whole lifecycle of a resource. We can put this under a separate section to make it easier to understand.

Reply:
Okay, I'll work on creating a section for this. Should it be part of the same guide though?